shinydashboard/lantern/functions/endpoints.R

```diff
@@ -111,3 +116,27 @@ get_vendor_list <- function(endpoint_export_tbl) {
 
   vendor_list <- c(vendor_list, vl)
 }
+
+# iterate over capability statements and extract table of resource types
+get_fhir_resources_tbl <- function(db_tables) {
+  fei <- db_tables$fhir_endpoints_info %>% collect() 
+  res <- fei %>%
+    purrr::pmap_dfr(function(...) {
+      current <- tibble(...)
+      cs <- jsonlite::fromJSON(fei %>% filter(id==current$id) %>% .$capability_statement) 
+      if (!is.null(cs)) {
+        resources <- purrr::pluck(cs,"rest","resource",1)
```

Comment:
This can be simplified to use the supported_resources array that is saved on each fhir_endpoint_info entry